circleguard/visualizer.py

```diff
@@ -517,7 +519,11 @@ def __init__(self, replays=[], beatmap_path=""):
         self.slider.setFixedHeight(20)
         self.slider.setStyleSheet("outline: none;")
         self.renderer.update_signal.connect(self.update_slider)
-        self.slider.valueChanged.connect(self.renderer.seek_to)
+        # don't want to use valueChanged because we change the value
+        # programatically and valueChanged would cuase a feedback loop.
+        # sliderMoved only activated on true user action, where we actually
```

Comment:
```suggestion
        # sliderMoved only activates on true user action, when we actually
```
unsure about the where/when part